docs/intro/tutorial.rst

```diff
@@ -138,7 +139,7 @@ To put our spider to work, go to the project's top level directory and run::
 
    scrapy crawl quotes
 
-This command runs the spider with name ``quotes`` that we've just added, that
+This command runs the spider with the name ``quotes`` that we've just added, that
```

Comment:
I’m not sure this “the” was necessary, although I am not 100% sure. It also feels like it adds potential for confusion, i.e. I think before it was clear that what was added was the spider, but with that “the”, it feels like it could be misinterpreted as “runs the spider whose name we have just added”, instead of “runs the spider named X that we have just added”.

Reply:
The original sentence without the "the" is grammatically incorrect. It could also be phrased "This command runs the spider named 'quotes' ". Would that be better?